typescript/sdk/src/token/deploy.ts

```diff
@@ -133,6 +150,20 @@ abstract class TokenDeployer<
 
     if (isCollateralTokenConfig(config) || isXERC20TokenConfig(config)) {
       return [config.token, scale, config.mailbox];
+    } else if (isEverclearCollateralTokenConfig(config)) {
+      return [
+        config.token,
+        scale,
+        config.mailbox,
+        config.everclearBridgeAddress,
+      ];
+    } else if (isEverclearEthBridgeTokenConfig(config)) {
+      return [
+        config.wethAddress,
+        scale,
```

Comment:
I believe this is removed from the constructor now
this highlights that we dont really have type safety on this code